packages/salesforcedx-vscode-core/src/commands/util/sourceResultOutput.ts

```diff
@@ -0,0 +1,245 @@
+/*
```

Comment:
Consolidated the beta deploy/retrieve output parsing to this file and renamed their respective functions to be consistent.